doc/source/tokudb/tokudb_troubleshooting.rst

```diff
@@ -0,0 +1,1678 @@
+
+**logger: next LSN**
+ This is the next unassigned Log Sequence Number. It will be assigned to the next entry in the recovery log.
+
+**logger: number of long logger write operations**
+ Number of times a logger write operation required 100ms or more.
+
+**logger: writes (bytes)**
+ Number of bytes the logger has written to disk.
+
+**logger: writes**
+ Number of times the logger has written to disk.
+
+**logger: writes (seconds)**
+ Number of seconds waiting for IO when writing logs to disk.
+
+**logger: writes (uncompressed bytes)**
+ Number of uncompressed the logger has written to disk.
+
+**max open dbs**
+ Max number of simultaneously open DBs.
+
+**memory: estimated maximum memory footprint**
+ Maximum memory footprint of the storage engine, the max value of (used - freed).
+
+**memory: largest attempted allocation size**
+ Largest number of bytes in a single successful malloc() operation.
+
+**memory: mallocator version**
+ Version string from in-use memory allocator.
+
+**memory: mmap threshold**
+ The threshold for malloc to use mmap.
+
+**memory: number of bytes freed**
+ Total number of mallocated bytes freed (used - freed = bytes in use).
+
+**memory: number of bytes requested**
+ Total number of bytes requested from mallocator.
+
+**memory: number of bytes used (requested + overhead)**
+ Total number of bytes allocated by mallocator.
+
+**memory: number of free operations**
+ Number of calls to free().
+
+**memory: number of malloc operations**
+Number of calls to malloc().
+
+**memory: number of malloc operations that failed**
+ Number of failed calls to malloc().
+
+**memory: number of realloc operations**
+ Number of calls to realloc().
+
+**memory: number of realloc operations that failed**
+ Number of failed calls to realloc().
+
+**memory: size of the last failed allocation attempt**
+ Largest number of bytes in a single failed malloc() operation.
+
+**num open dbs now**
+ Number of currently open DBs.
+
+**period, in ms, that recovery log is automatically fsynced**
+ ``fsync()`` frequency in milliseconds.
+
+**time now**
+ Current date/time on server.
+
+**time of engine startup**
+ This is the time when the |TokuDB| storage engine started up. Normally, this is when ``mysqld`` started.
+
+**time of environment creation**
+ This is the time when the |TokuDB| storage engine was first started up. Normally, this is when ``mysqld`` was initially installed with |TokuDB| 5.x. If the environment was upgraded from |TokuDB| 4.x (4.2.0 or later), then this will be displayed as "Dec 31, 1969" on Linux hosts.
+
+**txn: aborts**
+ This is the total number of transactions that have been aborted.
+
+**txn: begin**
+ This is the number of transactions that have been started.
+
+**txn: begin read only**
+ Number of read only transactions started.
+
+**txn: successful commits**
+ This is the total number of transactions that have been committed.
+
+Global Status
```

Comment:
I've added the missing status/variables. Also left a note on FT-674 due to duplicates.